src/styles/ParentStyles/dashboard.less

```diff
@@ -0,0 +1,19 @@
+.dashboard-container{
+    // display:flex;
+}
+
+.dashboard-content{
+    display:flex;
+    flex-direction: row;
+}
+
+.column{
+    border: 1px solid red;
+}
+
+.parent-dashboard-calendar-card{
+    width: 20rem;
+    max-height: 20rem;
+    border: 1px solid #f0f0f0;
+    border-radius: none;
+}
```

Comment:
I am not sure, but I believe this 'no new line at the end of file' warning can be removed by adding an extra blank line.